gpg_key.go

```diff
@@ -66,6 +83,30 @@ type GPGKeyUpdateOptions struct {
 	Namespace string `jsonapi:"attr,namespace"`
 }
 
+func (s *gpgKeys) List(ctx context.Context, registryName RegistryName, options *GPGKeyListOptions) (*GPGKeyList, error) {
+	if registryName != PrivateRegistry {
+		return nil, ErrInvalidRegistryName
+	}
```

Comment:
I don't find this very user friendly since it must always be "private". Another alternative is to omit the parameter and allow extension in the future by naming the method "ListPrivate". WDYT?

Reply:
Cool with removing this param and renaming to ListPrivate (on the off chance we end up supporting listing keys from the public registry).